src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -135,7 +135,7 @@
       :sortable-columns? true
       :always-sort? false
       :filterable? true
-      :cell-padding-left "16px"
+      :cell-padding-left 16
```

Comment:
Ugh, this is totally my bad for this sucking in this way, but this one needs to be the "16px" value.  I'll hopefully be overhauling how tables are defined shortly and that will get rid of this stupidity.